.gitignore

```diff
@@ -33,4 +33,5 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
-.env
\ No newline at end of file
+.env
+yarn.lock
```

Comment:
we should keep yarn.lock in the repository https://yarnpkg.com/getting-started/qa#should-lockfiles-be-committed-to-the-repository